test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -1182,6 +1183,27 @@ def sym_scatter_max(g, src, index, dim, out, dim_size):
                 do_constant_folding=True,
             )
 
+    @common_utils.parametrize("fp8_dtype", [torch.float8_e4m3fn, torch.float8_e5m2])
+    def test_fp8_export(self, fp8_dtype: torch.dtype):
```

Comment:
This test is tricky.

Technically speaking, fp8 support depends on onnx opset 19 (opset 19 introduced fp8 types to onnx), so this test should be `@skipIfUnsupportedMinOpsetVersion(19)`

On the other hand, it seems we are capping the old exporter to 17 (although it should be 18, as col2im was specified and implemented at opset 18), so this test should never be executed in the torch script exporter

We should probably port this test for the dynamo exporter at `test/onnx/test_fx_to_onnx_with_onnxruntime.py` or remove the 17/18 cap?

Reply:
I think it's ok as-is since we don't parameterize this test class on opset version - there's no point skipping.